src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +51,170 @@ export const KafkaPage: FunctionComponent = () => {
+                      <DescriptionListDescription>
+                        {"$0.09/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                    <DescriptionListGroup>
+                      <DescriptionListTerm>{t("storage")}</DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$0.10/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                  </DescriptionList>
+                </FlexItem>
+              </Flex>
+              <Divider isVertical />
+              <FlexItem flex={{ default: "flex_1" }}>
+                <TextContent>
+                  <Text component={TextVariants.p}>
+                    <Trans
+                      i18nKey={"kafkaoverview:streamingUnitText"}
+                      values={{
+                        value: "streaming unit",
+                      }}
+                    />
+                  </Text>
+                </TextContent>
+              </FlexItem>
+            </Flex>
+          </CardBody>
+        </Card>
+      </MarketingPageSection>
+      <MarketingPageSection>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h3">{t("kafkaInstanceCapacityTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Stack hasGutter>
+              <Text component={TextVariants.p}>
+                {t("kafkaInstanceCapacityDescription")}
+              </Text>
+              <KafkaInstanceCapacityTable />
+            </Stack>
+          </CardBody>
+          <CardFooter>
+            <TextContent>
+              <Text className={"pf-u-font-size-sm"}>
+                {
+                  <Trans
+                    i18nKey={"kafkaoverview:kafkaInstanceCapacityfooter"}
+                    components={{
+                      support: (
+                        <a
+                          href="https://access.redhat.com/support"
+                          target="_blank"
+                        />
+                      ),
+                      value: (
+                        <a
+                          href="https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/aced8e5e-8229-4cb2-82f9-87a8caa24bb3"
+                          target="_blank"
+                        />
+                      ),
+                    }}
+                  />
+                }
+              </Text>
+            </TextContent>
+          </CardFooter>
+        </Card>
+      </MarketingPageSection>
+      <MarketingPageSection>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h3">{t("cloudProvidersTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Stack hasGutter>
+              <Text component={TextVariants.p}>
+                {t("cloudProvidersDescription")}
+              </Text>
```

Comment:
With the content changes to make this line shorter (i.e. "Currently available"), we should also modify the layout so that this text is centered with the logo. 

This same treatment would be applied to the Azure logo when you add it. 

I'm double-checking with visual design to see if additional styles should be applied, but I might not hear back right away.